tests/Integration/Controller/AbstractControllerTest.php

```diff
@@ -102,6 +102,16 @@ protected function getDecodedJsonResponseContent(): array
         return json_decode($this->client->getResponse()->getContent(), true);
     }
 
+    /**
+     * Returns Int as response content.
```

Comment:
Can we make the intent clearer with the description and the name? I suggest `getResponseContentAsInt`, and "Returns the response content as int.".